sousmot/sousmot/settings.py

```diff
@@ -55,7 +55,7 @@
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
+        'DIRS': [BASE_DIR / "templates"],
```

Comment:
```suggestion
        'DIRS': [],
```
Is it really needed ? I didn't have problems on my side